tardis/interfaces/siteadapter.py

```diff
@@ -104,6 +105,13 @@ def drone_environment(
 
         return drone_environment
 
+    @property
+    def drone_heartbeat_interval(self) -> int:
```

Comment:
I would expect a short docstring here 😇

Reply:
Damn! I simply forgot to add it. This is not a big deal. I will add now.